tokens/nfd.json

```diff
@@ -0,0 +1,10 @@
U5MSAxNy4wOTMyWiIgZmlsbD0iI0VCNTc1NyIgc3Ryb2tlPSIjRUI1NzU3IiBzdHJva2Utd2lkdGg9IjAuNDc4MTMzIi8+CjxwYXRoIGQ9Ik01OS4yMzExIDc0LjM0OTZDNTcuMjc4NyA2OC4wNTQyIDUzLjQyMTggNDkuNjU0MSA1My42MTMgMjYuNDE2OEM1My42NTI5IDI1LjgxOTIgNTQuNTQ1NCAyNC4xMjE4IDU3Ljc5NjcgMjIuMTEzNkM1OC44NzI1IDIxLjQ0OTIgNjMuODI1MiAyMS43NTUgNjQuNjM2IDIwLjU1OTciIHN0cm9rZT0iIzBENEQ5MCIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIi8+CjxwYXRoIGQ9Ik01OC4xNDE5IDIxLjM4MTdMNTguMzYxMSAxOS4zNTY1QzU4LjM2MTEgMTkuMzU2NSA1OS4xNTEgMTkuNjcgNTkuNjEyIDIwLjY3NzNDNjAuMDcyOSAyMS42ODQ3IDU5LjYxMiAyMS42MzU0IDU5LjYxMiAyMS42MzU0TDU4LjE0MTkgMjEuMzgxN1oiIGZpbGw9IiNGRkVDREQiLz4KPHBhdGggZD0iTTU5LjA0MTggMTMuNDk4Mkw1OC44MjI2IDE1LjUyMzRDNTguODIyNiAxNS41MjM0IDU4LjAzMjcgMTUuMjA5OSA1Ny41NzE4IDE0LjIwMjVDNTcuMTEwOSAxMy4xOTUxIDU3LjU3MTggMTMuMjQ0NCA1Ny41NzE4IDEzLjI0NDRMNTkuMDQxOCAxMy40OTgyWiIgZmlsbD0iI0ZGRUNERCIvPgo8cGF0aCBkPSJNNjAuMzM4MiAxMy4yNTkyTDYwLjU1NzQgMTUuMjg0NEM2MC41NTc0IDE1LjI4NDQgNjEuMzQ3MyAxNC45NzA5IDYxLjgwODIgMTMuOTYzNUM2Mi4yNjkxIDEyLjk1NjEgNjIuMzM5IDExLjM1NTcgNjIuMzM5IDExLjM1NTdMNjAuMzM4MiAxMy4yNTkyWiIgZmlsbD0iI0ZGRUNERCIvPgo8cGF0aCBkPSJNNjAuMDkzOCAyMS42MzU1QzU3LjYyMzQgMjEuNjc1MyA1My4zODU5IDIxLjA2MTcgNTIuMDQ3MSAxOC4yODg2QzUxLjE5MjcgMTYuNTE4NyA1NC41NzMyIDE0Ljc5MjcgNTcuNjY1MiAxNC4yMjQ0QzU3LjY2NTIgMTUuMDg1MSA1OC40NzAxIDE1LjQ1OTYgNTguODcyNSAxNS41MzkzTDU5LjMzODYgMTMuNzQ2M0M1OS42MjE1IDEzLjcwNjUgNjAuMjExMyAxMy42MDI5IDYwLjMwNjkgMTMuNTA3MkM2MC40MDI1IDEzLjQxMTYgNjAuNTQ2IDEzLjc0NjMgNjAuNzg1IDE1LjE4MDdDNjEuOTgwNCAxNC43MDI2IDYzLjI0NzQgMTAuODc3NSA2Mi4wOTk5IDkuNDQzMTIiIHN0cm9rZT0iIzBENEQ5MCIvPgo8cGF0aCBkPSJNNTMuMDUzNCA3NC43NjU5QzUwLjM4NjcgNzEuMjY1OSA0NS4zNTM0IDYxLjU2NTkgNDYuNTUzNCA1MC43NjU5IiBzdHJva2U9IiMwRDREOTAiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIvPgo8cGF0aCBkPSJNMzEuMzcxNyA1NC4yNjU5QzM0LjAzODQgNTYuOTMyNSAzNy44NzE3IDYzLjk2NTkgMzEuODcxNyA3MC43NjU5QzMzLjUzODQgNzAuNzY1OSAzNy42NzE3IDcxLjU2NTkgMzguODcxNyA3NC43NjU5IiBzdHJva2U9IiMwRDREOTAiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIvPgo8cGF0aCBkPSJNNTQuMzcxNiA3Mi43NjU5QzU0LjM3MTYgNzMuMDk5MiA1NC41NzE2IDczLjk2NTkgNTUuMzcxNiA3NC43NjU5IiBzdHJva2U9IiMwRDREOTAiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIvPgo8cGF0aCBkPSJNNTYuMzcxNiA3Mi43NjU5QzU2LjM3MTYgNzMuMDk5MiA1Ni41NzE2IDczLjk2NTkgNTcuMzcxNiA3NC43NjU5IiBzdHJva2U9IiMwRDREOTAiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIvPgo8cGF0aCBkPSJNMzIuMzcxNiA3My43NjU5QzMyLjg3MTYgNzMuNzY1OSAzMy45NzE2IDc0LjA2NTkgMzQuMzcxNiA3NS4yNjU5IiBzdHJva2U9IiMwRDREOTAiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIvPgo8cGF0aCBkPSJNMzUuMzcxNiA3My43NjU5QzM1LjcwNDkgNzMuNzY1OSAzNi41NzE2IDc0LjA2NTkgMzcuMzcxNiA3NS4yNjU5IiBzdHJva2U9IiMwRDREOTAiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIvPgo8cGF0aCBkPSJNMzIuODcxNyA2Ljc2NTg3QzMyLjg3MTcgNy45MzI1NCAzMy4zNzE3IDEwLjY2NTkgMzUuMzcxNyAxMi4yNjU5IiBzdHJva2U9IiMwRDREOTAiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIvPgo8cGF0aCBkPSJNNDUuMzcxNyAzLjI2NTg3QzQ0LjcwNSA0LjQzMjU0IDQzLjY3MTcgNy4xNjU4NyA0NC44NzE3IDguNzY1ODciIHN0cm9rZT0iIzBENEQ5MCIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIi8+CjxwYXRoIGQ9Ik00Ni44NzE3IDExLjc2NTlDNDcuODcxNyAxMi4yNjU5IDUwLjA3MTcgMTIuNjY1OSA1MC44NzE3IDEwLjI2NTkiIHN0cm9rZT0iIzBENEQ5MCIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIi8+Cjwvc3ZnPgo=",
```

Comment:
It's better to use an optimized URL. Please use something like this [tool](https://yoksel.github.io/url-encoder/)

Reply:
URL Updated